gcore/src/msg.rs

```diff
@@ -221,6 +233,59 @@ pub fn reply(payload: &[u8], value: u128) -> Result<MessageId, ReplyError> {
     }
 }
 
+/// Send a new message with gas limit as a reply to the message currently being
```

Comment:
Seems better just to refer gasless function that this is the same, but with concrete gas amount